agents/xy_stage/xy_ocs_agent.py

```diff
@@ -0,0 +1,249 @@
+import os
+import argparse
+import time
+import txaio
+
+import numpy as np
+
+## yes I shouldn't have named that module agent
+from xy_agent.xy_connect import XY_Stage
```

Comment:
Also, where is this driver located? Any chance we can put it in the `socs/agents` directory?

Reply:
Drivers are here: https://github.com/kmharrington/xy_stage_control 

I don't want to add the client side of the drivers to socs because I need to be able to use it outside ocs as well. And the would require maintaining two sets of code that do the exact same thing.